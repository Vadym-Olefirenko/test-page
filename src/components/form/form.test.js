import React from 'react';
import Form from './form.js';
import TextArea from '../textArea/textarea';
import {create, act} from 'react-test-renderer';
import { configure, shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });


describe('Testing Form', () => {
    test('Form rendered', () => {
        const FormTest = mount(<Form/>).debug();
        expect(FormTest).toMatchSnapshot();
    });
});





