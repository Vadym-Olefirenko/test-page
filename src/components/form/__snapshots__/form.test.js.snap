// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Form Form rendered 1`] = `
"<Form>
  <form className=\\"form\\" onSubmit={[Function: handleSubmit]}>
    <Row tag=\\"div\\" widths={{...}}>
      <div className=\\"row\\">
        <Col lg=\\"9\\" tag=\\"div\\" widths={{...}}>
          <div className=\\"col-lg-9\\">
            <div className=\\"form__left\\">
              <h3>
                ЗАМОВИТИ РЕДАГУВАННЯ
              </h3>
              <p className=\\"form__desk\\">
                Виправимо всі помилки, приберемо всі дурниці, перефразуємо невдалі місця, але сильно текст
                <br />
                 не переписуватимемо. Зайвих виправлень не буде. Детальніше про редагування
              </p>
              <div className=\\"input__row required\\">
                <input type=\\"text\\" name=\\"email\\" value=\\"\\" onChange={[Function: onChange]} placeholder=\\"Ваша эл.почта\\" />
              </div>
              <div className=\\"input__row\\">
                <input type=\\"text\\" name=\\"name\\" value=\\"\\" onChange={[Function: onChange]} placeholder=\\"Ваше имя\\" />
              </div>
              <div className=\\"input__row area__row\\">
                <TextArea value=\\"\\" textAreaValue={[Function: textAreaValue]}>
                  <textarea value=\\"\\" onChange={[Function: textAreaValue]} name=\\"text\\" placeholder=\\"Введіть текст або  \\" />
                </TextArea>
                <FileLoad file={[Function: readFile]}>
                  <label className=\\"file__loader\\" htmlFor=\\"file\\">
                    завантажте файл
                    <input id=\\"file\\" type=\\"file\\" name=\\"file\\" onChange={[Function: readFile]} placeholder=\\"Ваше имя\\" accept=\\".doc,.docx,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,.csv, application/vnd.openxmlformats-officedocument.spreadsheetml.sheet, application/vnd.ms-excel,application/msword, application/vnd.ms-excel, application/vnd.ms-powerpoint, text/plain, application/pdf, .rtf, .txt, .pdf, .zip\\" />
                  </label>
                </FileLoad>
                <div className=\\"symbols__value\\" id=\\"symbols__value\\" />
              </div>
              <div className=\\"lang__row\\">
                <h3>
                  МОВА
                </h3>
                <div className=\\"radioBtn__item\\">
                  <input type=\\"radio\\" id=\\"ua\\" name=\\"language\\" value=\\"ua\\" onClick={[Function: langValue]} />
                  <label htmlFor=\\"ua\\">
                    Українська
                  </label>
                </div>
                <div className=\\"radioBtn__item\\">
                  <input type=\\"radio\\" id=\\"ru\\" name=\\"language\\" value=\\"ru\\" onClick={[Function: langValue]} />
                  <label htmlFor=\\"ru\\">
                    Русский
                  </label>
                </div>
                <div className=\\"radioBtn__item\\">
                  <input type=\\"radio\\" id=\\"en\\" name=\\"language\\" value=\\"en\\" onClick={[Function: langValue]} />
                  <label htmlFor=\\"en\\">
                    English
                  </label>
                </div>
              </div>
              <div className=\\"input__row\\">
                <input type=\\"text\\" name=\\"comment\\" value=\\"\\" onChange={[Function: onChange]} placeholder=\\"Стислий коментар\\" />
              </div>
            </div>
          </div>
        </Col>
        <Col lg=\\"3\\" tag=\\"div\\" widths={{...}}>
          <div className=\\"col-lg-3\\">
            <div className=\\"form__right\\">
              <div className=\\"price__content\\">
                <div className=\\"price\\">
                  0.00 грн
                </div>
                <div className=\\"time\\" />
                <button type=\\"submit\\" className=\\"order__btn\\">
                  Замовити
                </button>
              </div>
            </div>
          </div>
        </Col>
      </div>
    </Row>
  </form>
</Form>"
`;
